src/stores/GlobalStore.js

```diff
@@ -37,6 +40,9 @@ export default class GlobalStore {
     },
   });
 
+  setOnline = () => this.online = true;
+  setOffline = () => this.online = false;
```

Comment:
nit: make a single function that handles both cases:
`toggleOnline = (online) => this.online = online;`